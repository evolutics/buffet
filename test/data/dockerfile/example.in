ARG example_foo='a'

FROM alpine AS example
ARG example
ARG example_bar='b'
LABEL org.opencontainers.image.title="Example"
RUN touch /root/example

FROM alpine
ARG example
ARG example_baz='c'

LABEL org.opencontainers.image.title="Example"
LABEL org.opencontainers.image.url="https://example.com"
LABEL info.evolutics.freezer.help="example --help"

COPY --from=example /root/example /usr/local/bin/
RUN ls
