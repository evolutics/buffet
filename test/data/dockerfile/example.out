ARG example=''
ARG example_foo='a'

ARG _alpine_version='3.9.4'

FROM alpine AS example
ARG example
ARG example_bar='b'
RUN if [[ -n "${example}" ]]; then \
    touch /root/example \
  ; fi

FROM alpine:"${_alpine_version}"

ARG example
ARG example_baz='c'
COPY --from=example /root/example* /var/empty /usr/local/bin/
RUN if [[ -n "${example}" ]]; then \
    ls \
  ; fi

WORKDIR /workdir
